zookeeper-server/src/main/java/org/apache/zookeeper/server/quorum/FastLeaderElection.java

```diff
@@ -1045,6 +1057,9 @@ public Vote lookForLeader() throws InterruptedException {
                         /*
                          * Before joining an established ensemble, verify that
                          * a majority are following the same leader.
+                         *
+                         * Note that the outofelection map also stores votes from the current leader election.
+                         * See ZOOKEEPER-1732 for more information.
```

Comment:
Is `ZOOKEEPER-1732` the right JIRA? That JIRA has no direct correlation with the comment added here, as far as I can tell.